BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/SkontoViewModel.swift

```diff
@@ -171,4 +186,20 @@ class SkontoViewModel {
                                 skontoDiscounts: nil,
                                 candidates: skontoDiscounts.initialExtractionResult.candidates)
     }
+
+    private func determineSkontoEdgeCase() {
+        if remainingDays < 0 {
```

Comment:
[suggestion]Can you please add a comment why the order is important in this method?